tests/unittests/functions/report/test_firewatch_functions_report_add_label_to_ticket_when_job_run_passes.py

```diff
@@ -0,0 +1,79 @@
+import pytest
+import simple_logger.logger
+from jira import Issue
+
+from cli.report import Report
+from cli.report.constants import JOB_PASSED_SINCE_TICKET_CREATED_LABEL
+
+_logger = simple_logger.logger.get_logger(__name__)
+
+
+@pytest.fixture
+def job(patch_job_junit_dir, patch_job_log_dir, default_jira_project, job):
+    yield job
+
+
+@pytest.fixture
+def firewatch_config(firewatch_config_json, default_jira_project, firewatch_config):
+    yield firewatch_config
+
+
+@pytest.fixture
+def report(firewatch_config, job):
+    yield Report(firewatch_config=firewatch_config, job=job)
+
+
+@pytest.fixture
+def fake_job_has_open_bugs(monkeypatch, fake_issue_key):
+    def _get_open_bugs(*args, **kwargs):
+        _logger.info("Patching Report._get_open_bugs")
+        return [fake_issue_key]
+
+    monkeypatch.setattr(Report, "_get_open_bugs", _get_open_bugs)
+
+
+def test_fixtures_fake_jira_issue_exists(
+    patch_jira_api_requests,
+    jira,
+    fake_issue_id,
+    fake_issue_key,
+):
+    issue = jira.get_issue_by_id_or_key(fake_issue_id)
+    assert isinstance(issue, Issue)
```

Comment:
`assert isinstance(issue, Issue)` this will check nothing but that issue, Issue which will be always True

Reply:
Yes, that is correct. The return type of `jira.get_issue_by_id_or_key` _should_ be (but is not guaranteed to be) an instance of Issue, for the purpose of this test. If that ever changes (during refactoring, for example), that will be a signal that this test should be updated.

Reply:
ok